sphinxcontrib/openapi/openapi30.py

```diff
@@ -209,8 +215,10 @@ def _example(media_type_objects, method=None, endpoint=None, status=None,
                     .format(**locals())
                 yield '{extra_indent}{indent}Host: example.com' \
                     .format(**locals())
-                yield '{extra_indent}{indent}Content-Type: {content_type}' \
-                    .format(**locals())
+                if content_type:
+                    yield \
```

Comment:
nit: Can't this fit on one line?

Reply:
Yes, but without a space before the backspace.